lib/google_maps.ex

```diff
@@ -1523,6 +1523,72 @@ defmodule GoogleMaps do
+    Photo references are returned from either a Place Search or Place Details request.
+
+  * `dimension_constraint` — A tuple of the form `{:maxheight, integer}` or
+    `{:maxwidth, integer}`. Specifies the maximum desired height or width, in pixels,
+    of the image returned by the Place Photos service. If the image is smaller than the
+    values specified, the original image will be returned. If the image is larger in
+    either dimension, it will be scaled to match the smaller of the two dimensions,
+    restricted to its original aspect ratio. Both the maxheight and maxwidth properties
+    accept an integer between 1 and 1600.
+
+  ## Options:
+    While no optional parameters for Place Photos exist at the time of writing, the
+    options keyword list supports the `key` parameter in case an explicit API key
+    is to be supplied.
+
+  ## Returns
+
+    This function returns `{:ok, body}` if the request is successful.
+    The returned body is a binary containing jpeg image data.
+
+    If the API key is invalid or the request has exceeded your quota, the function
+    returns `{:error, 403}`.
+
+    Otherwise, if the request is invalid, the return value will be `{:error, 400}`.
+    This likely indicates an incorrect photo reference, but can also mean an
+    invalid value for `maxwidth` or `maxheight`.
+
+  ## Examples
+
+      # Request with an invalid API key
+      iex> photo_reference = "CnRtAAAATLZNl354RwP_9UKbQ_5Psy40texXePv4oAlgP4qNEkdIrkyse7rPXYGd9D_Uj1rVsQdWT4oRz4QrYAJNpFX7rzqqMlZw2h2E2y5IKMUZ7ouD_SlcHxYq1yL4KbKUv3qtWgTK0A6QbGh87GB3sscrHRIQiG2RrmU_jF4tENr9wGS_YxoUSSDrYjWmrNfeEHSGSc3FyhNLlBU"
+      iex> GoogleMaps.place_photo(photo_reference, {:maxwidth, 500}, key: "invalid key")
+      {:error, 403}
+
+      # Request with an invalid photo reference
+      iex> GoogleMaps.place_photo("invalid reference", {:maxwidth, 500})
+      {:error, 400}
+
+      # Request a photo with a maximum height of 100
+      iex> photo_reference = "CnRtAAAATLZNl354RwP_9UKbQ_5Psy40texXePv4oAlgP4qNEkdIrkyse7rPXYGd9D_Uj1rVsQdWT4oRz4QrYAJNpFX7rzqqMlZw2h2E2y5IKMUZ7ouD_SlcHxYq1yL4KbKUv3qtWgTK0A6QbGh87GB3sscrHRIQiG2RrmU_jF4tENr9wGS_YxoUSSDrYjWmrNfeEHSGSc3FyhNLlBU"
+      iex> {:ok, response} = GoogleMaps.place_photo(photo_reference, {:maxheight, 100})
+      iex> is_binary(response)
+      true
+  """
+  @spec place_photo(String.t, {:maxwidth, integer()} | {:maxheight, integer()}, options()) :: Response.t()
+  def place_photo(photo_reference, {constraint, value} = dimension_constraint, options \\ [])
+  when is_binary(photo_reference) and constraint in [:maxwidth, :maxheight] and is_integer(value) do
+    params = Keyword.merge(options, [dimension_constraint,
+                                     {:resource, "photo"},
+                                     {:photoreference, photo_reference},
+                                     {:options, [follow_redirect: true]}])
+
+    GoogleMaps.get("place", params)
```

Comment:
I would rather use `GoogleMaps.get("place/photo", params)` to indicate the resource, and to conform with "place/details".

The `resource` param may need to be changed to something else, such as `output` (to be the same as Google), and it is default to `json`.

In this particular case, `{:output, ""}` will be set in `params`.

Reply:
Alright, sounds good!